.github/CODEOWNERS

```diff
@@ -6,5 +6,6 @@ docs/developer-docs/build/agents @dfinity/dx
 docs/developer-docs/build/cdks @dfinity/dx
 docs/developer-docs/integrations/rosetta @dfinity/finint
 docs/developer-docs/security @dfinity/product-security
+docs/references/execution-errors.mdx @dfinity/runtime @dfinity/execution
```

Comment:
"Unknown owner: make sure that the team @dfinity/runtime and @dfinity/execution exist, are publicly visible, and has write access to the repository".

Reply:
Any idea who has control of that? The teams seem to exist and be visible based on [this search](https://github.com/orgs/dfinity/teams?page=1&query=+visibility%3Avisible), so I guess the issue is that they don't have write access to the rep?

Reply:
Yea it looks like they need write access to the repo. I'm not sure who can provide that - IT maybe?